table_evaluator/table_evaluator.py

```diff
@@ -474,9 +493,11 @@ def estimator_evaluation(self, target_col: str, target_type: str = 'class') -> f
         for estimator in self.estimators:
             assert hasattr(estimator, 'fit')
             assert hasattr(estimator, 'score')
-
+        
         self.fit_estimators()
         self.estimators_scores = self.score_estimators()
+        print("F1-scores and their Jaccard similarity:")
+        print(self.scores_df)
```

Comment:
The scores_df should be the new estimator_scores imo. We should remove the old estimator_scores and use this as the new one. Furthermore, we should integrate the score from these scores into the similarity score, similar to what the old estimator_scores was doing. 

Can you propose a way to integrate this nicely? Does using the 1-MAPE/MSE still make sense for these cases?